itunesCall.js

```diff
@@ -0,0 +1,23 @@
+const fetch = require("node-fetch");
+
+const getPodcastData = async () => {
+    try {
+		const response = await fetch(`https://itunes.apple.com/search?term=$podcast`, {
+			"method": "GET",
+		})
+		const resolved = await response.json()
+		return resolved
+	}
+	catch (e) {
+		console.log(e);
+	}
+}
+
+const getAllPodcasts = async () => {
+    const data = await getPodcastData()
+    return data
+  }
```

Comment:
Is there a reason why this is split up like this? Seems to me like we could just invoke getPodcastData() outright

Reply:
I'm wondering the same thing.  What purpose does `getAllPodcasts()` serve?

Reply:
I think you are right, this is an unnecessary step, e can definitely refactor this once we put everything together.  As of right now, I created this file to get by but I think i will make more sense to have all API call in one single folder/file